reference/excel/chartformatstring.md

```diff
@@ -1,6 +1,6 @@
 # ChartFormatString Object (JavaScript API for Excel)
 
-Represents the characters in chart object that contains text, like chart title, chart axis title, etc.
+Represents the substring in chart related objects that contains text, like ChartTitle object, ChartAxisTitle object, etc.
```

Comment:
Please avoid using "etc." as that is ambiguous. You can change "like" to "for example" here and remove etc.